### ISSUE TITLE
Cultists

### ISSUE THREAD

**Issue by DarkSkyT9A** [BSData/The-9th-Age#997]
[Cult Leader](https://www.the-ninth-age.com/community/lexicon/index.php?entry/4553-cult-leader/)
-When Army General and Master of Ritual is selected, incorrectly adds [Veil Walker](https://www.the-ninth-age.com/community/lexicon/index.php?entry/690-veil-walker/) as an option. [Veil Walker](https://www.the-ninth-age.com/community/lexicon/index.php?entry/690-veil-walker/) is not part of this faction (Probably used to be, isn't now).

Daemon Symbiote:
-Manifestations with [Dominant](https://www.the-ninth-age.com/community/lexicon/index.php?entry/4602-dominant/) don't seem to be tracked with the other [dominant](https://www.the-ninth-age.com/community/lexicon/index.php?entry/4602-dominant/) special items. Player shouldn't be able to have more than 1 [dominant](https://www.the-ninth-age.com/community/lexicon/index.php?entry/4602-dominant/) special item, of which manifestations are special items. list of [dominant](https://www.the-ninth-age.com/community/lexicon/index.php?entry/4602-dominant/) items daemon symbiotic has access to:
->[Broodmother](https://www.the-ninth-age.com/community/lexicon/index.php?entry/1852-broodmother/) (Manifestation, Cultists Book)
->[Mesmerising Plumage](https://www.the-ninth-age.com/community/lexicon/index.php?entry/1054-mesmerising-plumage/) (Manifestation, Cultists Book)
->[Chilling Yawn](https://www.the-ninth-age.com/community/lexicon/index.php?entry/1895-chilling-yawn/) (Manifestation, Cultists Book)
->[Aura of Despair](https://www.the-ninth-age.com/community/lexicon/index.php?entry/1896-aura-of-despair/) (Manifestation, Cultists Book)
->[Ledger of Souls](https://www.the-ninth-age.com/community/lexicon/index.php?entry/683-ledger-of-souls/) ([Artefact](https://www.the-ninth-age.com/community/lexicon/index.php?entry/1655-artefact/), Cultists Book)
->Lifeforce Talisman ([Artefact](https://www.the-ninth-age.com/community/lexicon/index.php?entry/1655-artefact/), Cultists Book)
->[Crystal Ball](https://www.the-ninth-age.com/community/lexicon/index.php?entry/4172-crystal-ball/) ([Artefact](https://www.the-ninth-age.com/community/lexicon/index.php?entry/1655-artefact/), Arcane Compendium)
->[Magical Heirloom](https://www.the-ninth-age.com/community/lexicon/index.php?entry/4169-magical-heirloom/) ([Artefact](https://www.the-ninth-age.com/community/lexicon/index.php?entry/1655-artefact/), Arcane Compendium)
->[Book of Arcane Mastery](https://www.the-ninth-age.com/community/lexicon/index.php?entry/4368-book-of-arcane-mastery/) ([Artefact](https://www.the-ninth-age.com/community/lexicon/index.php?entry/1655-artefact/), Arcane Compendium)

Summoned Daemons for Cultists have been altered in their book so they don't have access to the alternate deployment options when found in the [DL](https://www.the-ninth-age.com/community/lexicon/index.php?entry/4041-dl/&synonym=749) book because summoned Daemons can only be summoned to the battlefield.
->[Imps](https://www.the-ninth-age.com/community/lexicon/index.php?entry/1883-imps/) (Should not have [Dark Hide](https://www.the-ninth-age.com/community/lexicon/index.php?entry/1927-dark-hide/) Manifestation Access)
->[Threshing Engine](https://www.the-ninth-age.com/community/lexicon/index.php?entry/1879-threshing-engine/) (Should not have [Horns of Hubris](https://www.the-ninth-age.com/community/lexicon/index.php?entry/1894-horns-of-hubris/) Access)
->[Eidolons](https://www.the-ninth-age.com/community/lexicon/index.php?entry/1865-eidolons/) (Should not have option to take [Scout](https://www.the-ninth-age.com/community/lexicon/index.php?entry/102-scout/))
->[Hellhounds](https://www.the-ninth-age.com/community/lexicon/index.php?entry/1051-hellhounds/) (Should not have [Horns of Hubris](https://www.the-ninth-age.com/community/lexicon/index.php?entry/1894-horns-of-hubris/) Access)
->[Mageblight Gremlins](https://www.the-ninth-age.com/community/lexicon/index.php?entry/3798-mageblight-gremlins/) (Do not have [Veil](https://www.the-ninth-age.com/community/lexicon/index.php?entry/3990-veil/&synonym=700) Stalker)
->[Clawed Fiends](https://www.the-ninth-age.com/community/lexicon/index.php?entry/1055-clawed-fiends/) (Should not have [Horns of Hubris](https://www.the-ninth-age.com/community/lexicon/index.php?entry/1894-horns-of-hubris/) Access)
->[Sirens](https://www.the-ninth-age.com/community/lexicon/index.php?entry/1059-sirens/) (Do not have [Vanguard](https://www.the-ninth-age.com/community/lexicon/index.php?entry/213-vanguard/))
->[Blazing Glory](https://www.the-ninth-age.com/community/lexicon/index.php?entry/1899-blazing-glory/) (Should not have [Horns of Hubris](https://www.the-ninth-age.com/community/lexicon/index.php?entry/1894-horns-of-hubris/) Access)

Pledged to Darkness should read:
"[Models](https://www.the-ninth-age.com/community/lexicon/index.php?entry/81-models/&synonym=901) with Pledged to Darkness can only join or be joined by [models](https://www.the-ninth-age.com/community/lexicon/index.php?entry/81-models/&synonym=901) with Pledged to Darkness. While joined by
one or more Characters, [R&F](https://www.the-ninth-age.com/community/lexicon/index.php?entry/3952-r-amp-f/) [models](https://www.the-ninth-age.com/community/lexicon/index.php?entry/81-models/&synonym=901) with Pledged to Darkness gain [Fearless](https://www.the-ninth-age.com/community/lexicon/index.php?entry/4305-fearless/&synonym=848)."

Sacrificial Offerings (Cultists unit and Ritual Altar) should read:
"Immediately before performing a Casting Attempt with a [Wizard](https://www.the-ninth-age.com/community/lexicon/index.php?entry/4458-wizard/) in a unit containing one or more [models](https://www.the-ninth-age.com/community/lexicon/index.php?entry/81-models/&synonym=901) with
Sacrificial Offerings, you may inflict 1 hit against that unit. This hit wounds automatically with no saves of any kind
allowed. If a Health Point loss was caused against a [model](https://www.the-ninth-age.com/community/lexicon/index.php?entry/81-model/) with Sacrificial Offerings, the Casting Attempt gains a +1
Casting Modifier. Each unit can be affected by Sacrificial Offerings only once per Magic Phase."

[Cult Leader](https://www.the-ninth-age.com/community/lexicon/index.php?entry/4553-cult-leader/) is not [Fearless](https://www.the-ninth-age.com/community/lexicon/index.php?entry/4305-fearless/&synonym=848).

Cultists should not have [Frenzy](https://www.the-ninth-age.com/community/lexicon/index.php?entry/183-frenzy/).

Grand Council (Ritual Altar) should read:
"The [model](https://www.the-ninth-age.com/community/lexicon/index.php?entry/81-model/)’s Health Points and Attack Value are set to 8, and its base size is changed to 120 mm round. The [model](https://www.the-ninth-age.com/community/lexicon/index.php?entry/81-model/) can also select [Wrath of God](https://www.the-ninth-age.com/community/lexicon/index.php?entry/3871-wrath-of-god/) ([Thaumaturgy](https://www.the-ninth-age.com/community/lexicon/index.php?entry/3330-thaumaturgy/)) as one of its 2 spells."

Dark [Conclave](https://www.the-ninth-age.com/community/lexicon/index.php?entry/98-conclave/&synonym=278) (Ritual Altar) should read:
"Spells cast by the [model](https://www.the-ninth-age.com/community/lexicon/index.php?entry/81-model/) gain +3″ [range](https://www.the-ninth-age.com/community/lexicon/index.php?entry/4470-range/) for each additional friendly [Wizard](https://www.the-ninth-age.com/community/lexicon/index.php?entry/4458-wizard/) within 12″ of the [Caster](https://www.the-ninth-age.com/community/lexicon/index.php?entry/4462-caster/). This bonus can never increase the combined modifier beyond +9″, however other sources may.

The [model](https://www.the-ninth-age.com/community/lexicon/index.php?entry/81-model/) must select 2 spells from: [Spear of Infinity](https://www.the-ninth-age.com/community/lexicon/index.php?entry/1242-spear-of-infinity/) (Hereditary Spell), [Hand of Heaven](https://www.the-ninth-age.com/community/lexicon/index.php?entry/3866-hand-of-heaven/) ([Thaumaturgy](https://www.the-ninth-age.com/community/lexicon/index.php?entry/3330-thaumaturgy/)), and [The Grave Calls](https://www.the-ninth-age.com/community/lexicon/index.php?entry/4427-the-grave-calls/) [Occultism](https://www.the-ninth-age.com/community/lexicon/index.php?entry/3326-occultism/)) during Spell Selection. This overrides the normal Spell Selection rules connected to being a [Wizard Adept](https://www.the-ninth-age.com/community/lexicon/index.php?entry/4316-wizard-adept/).

When performing a Daemonic Summon with the [model](https://www.the-ninth-age.com/community/lexicon/index.php?entry/81-model/), divide the [chosen](https://www.the-ninth-age.com/community/lexicon/index.php?entry/2205-chosen/) unit’s Point Cost by 300 (instead of 200), rounding fractions up, for determining the number of discarded [Veil](https://www.the-ninth-age.com/community/lexicon/index.php?entry/3990-veil/&synonym=700) Tokens."
